src/App.js

```diff
@@ -1,25 +1,72 @@
 import logo from './logo.svg';
 import './App.css';
+import React, { useState } from 'react';
+import { getValue } from '@testing-library/user-event/dist/utils';
+import { useRef } from "react";
+export default App;
 
 function App() {
+
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [gender, setGender] = useState('');
+
+
+  const handleNameChange = (e) => {
+    setName(e.target.value);
+  }
+
+  const handleEmailChange = (e) => {
+    setEmail(e.target.value);
+  }
+
+  const handleGenderChange = (e) => {
+    setGender(e.target.value);
+  }
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log('Name:', name);
+    console.log('Email:', email);
+    console.log('Gender:', gender);
+  }
```

Comment:
good. but. we are using <form> only as a semantic html for all the crawlers and just to prove we are aware of it's existance. but. we don't want to use native solution of calling prop on this tag, which is 'onSubmit'. 

This function should be moved to 'button', ale type='submit' should be changed to 'button'. 

e.preventDefault() would not be needed then.